test/Calibur.execute.t.sol

```diff
@@ -319,7 +403,47 @@ contract CaliburExecuteTest is TokenHandler, HookHandler, ExecuteFixtures, Deleg
         uint256 nonce = key << 64 | seq;
 
         // Create hash of the calls + nonce and sign it
-        BatchedCall memory batchedCall = CallUtils.initBatchedCall().withCalls(calls).withShouldRevert(true);
+        BatchedCall memory batchedCall = CallUtils.initBatchedCall().withCalls(calls).withRevertOnFailure(true);
```

Comment:
We can do this in a sep PR but as I'm reviewing this noticing that we still have some opData language

Reply:
ah yes I'll do it here